crates/sozo/ops/src/events.rs

```diff
@@ -85,8 +86,10 @@ fn extract_events(
         events: &mut HashMap<String, Vec<Token>>,
         full_abi_path: &Utf8PathBuf,
     ) -> Result<()> {
-        let abi_str = fs::read_to_string(full_abi_path)?;
+        let abi_str = fs::read_to_string(full_abi_path)
+            .with_context(|| format!("Failed to read ABI file at path: {}", full_abi_path))?;
 
+        // TODO: add support for events emitted by world once its present in ABI
```

Comment:
Not sure what this is referring to? Aren't the event of the world already present here?

Or do you mean the event messages?

Reply:
Do we need an issue for this one @lambda-0x to track it?

Reply:
i meant the custom events defined by user, which is emitted using: https://github.com/dojoengine/dojo/blob/8dc5070a7862424ec258c632d8c99e6e25f0cb28/crates/dojo-core/src/world/world_contract.cairo#L747-L758

as we discussed since those `#[dojo::event]` are currently not annotated as events in the ABI, will open an issue for it.